docs/source/app-dev/bindings-scala/index.rst

```diff
@@ -0,0 +1,154 @@
+
+Create a contract and send a CreateCommand
+------------------------------------------
+
+To create a Scala class representing an **IOU** contract, you need the following **imports**:
+
+.. literalinclude:: ./code-snippets/quickstart-scala/application/src/main/scala/com/digitalasset/quickstart/iou/IouMain.scala
+   :start-after: // <doc-ref:imports>
+   :end-before: // </doc-ref:imports>
+
+the definition of the **issuer** ``Party``:
+
+.. literalinclude:: ./code-snippets/quickstart-scala/application/src/main/scala/com/digitalasset/quickstart/iou/IouMain.scala
+   :start-after: // <doc-ref:issuer-definition>
+   :end-before: // </doc-ref:issuer-definition>
+
+and the following code to create an instance of the ``M.Iou`` class:
+
+.. literalinclude:: ./code-snippets/quickstart-scala/application/src/main/scala/com/digitalasset/quickstart/iou/IouMain.scala
+   :start-after: // <doc-ref:iou-contract-instance>
+   :end-before: // </doc-ref:iou-contract-instance>
+
+To send a :ref:`com.digitalasset.ledger.api.v1.createcommand` (keep in mind the following code snippet is part of the Scala *for comprehension expression*):
+
+.. literalinclude:: ./code-snippets/quickstart-scala/application/src/main/scala/com/digitalasset/quickstart/iou/IouMain.scala
+   :start-after: // <doc-ref:submit-iou-create-command>
+   :end-before: // </doc-ref:submit-iou-create-command>
+
+For more details on how to submit a command, please refer to the implementation of `com.digitalasset.quickstart.iou.ClientUtil#submitCommand <https://github.com/digital-asset/daml/blob/master/language-support/scala/examples/quickstart-scala/application/src/main/scala/com/digitalasset/quickstart/iou/ClientUtil.scala>`_.
+
+Receive a transaction, exercise a choice and send an ExerciseCommand
+---------------------------------------------------------------------
+
+To receive a transaction as a **newOwner** and decode a :ref:`com.digitalasset.ledger.api.v1.createdevent` for ``IouTransfer`` contract, you need the definition of the **newOwner** ``Party``:
+
+.. literalinclude:: ./code-snippets/quickstart-scala/application/src/main/scala/com/digitalasset/quickstart/iou/IouMain.scala
+   :start-after: // <doc-ref:new-owner-definition>
+   :end-before: // </doc-ref:new-owner-definition>
+
+and the following code that handles subscription and decoding:
+
+.. literalinclude:: ./code-snippets/quickstart-scala/application/src/main/scala/com/digitalasset/quickstart/iou/IouMain.scala
+   :start-after: // <doc-ref:subscribe-and-decode-iou-transfer>
+   :end-before: // </doc-ref:subscribe-and-decode-iou-transfer>
+
+To exercise ``IouTransfer_Accept`` choice on the ``IourTransfer`` contract that you received and send a corresponding :ref:`com.digitalasset.ledger.api.v1.exercisecommand`:
```

Comment:
```suggestion
To exercise ``IouTransfer_Accept`` choice on the ``IouTransfer`` contract that you received and send a corresponding :ref:`com.digitalasset.ledger.api.v1.ExerciseCommand`:
```

Reply:
thanks... but the ref link is no caps.

Reply:
actually the doc build tool takes references in camel notation too and converts them to all small letters. I like camel... more readable. Going to check the rest of the link in the doc.